### ISSUE TITLE
jsonschema error

### ISSUE THREAD

**Issue by gdesmott** [flathub/flatpak-builder-lint#104]
I'm trying to update the runtime of the Karapulse package: https://github.com/flathub/org.karapulse.Karapulse/pull/16

Build failed here: https://buildbot.flathub.org/#/builders/19/builds/1575

```
{
    "errors": [
        "jsonschema-validation-error"
    ],
    "warnings": [
        "module-karapulse-source-sha1-deprecated",
        "finish-args-contains-both-x11-and-wayland"
    ],
    "jsonschema": [
        "{'dest': 'cargo/vendor/addr2line-0.13.0', 'url': 'data:%7B%22package%22%3A%20%221b6a2d3371669ab3ca9797670853d61402b03d0b4b9ebf33d677dfa720203072%22%2C%20%22files%22%3A%20%7B%7D%7D', 'dest-filename': '.cargo-checksum.json', 'type': 'file'} is not valid under any of the given schemas"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
```

The exact same manifest used to work fine, see https://github.com/flathub/org.karapulse.Karapulse/pull/15 , so I guess this may be a false positive?

**Comment by hfiguiere** [comment]
update the flatpak-builder tools and regenerate the cargo sources.

**Comment by gdesmott** [comment]
Thanks @hfiguiere !
